contracts/src/TroveManager.sol

```diff
@@ -1784,6 +1790,9 @@ contract TroveManager is LiquityBase, ITroveManager, ITroveEvents {
                 if (_batchDebt == 0) {
                     batchDebtSharesDelta = debtIncrease;
```

Comment:
So in this case, we're assuming the debt increase can't increase the shares ratio beyond the max right?

I guess that should hold since even if a batch's ratio got inflated, when the last Trove exited, the batch's shares should have been reset to 0.

Reply:
If the batch debt is zero, then the ratio debt / shares would either be zero or undefined. In the most common case, shares would also be zero, and after the update the ratio would be 1 (1e18).
Maybe we should explore if due to rounding, a batch that gets empty and then filled again, can have non zero shares and what are the implications, but here the ratio would deviate in the opposite direction, as shares would be bigger than debt.